cli/shared/src/main/scala/ch/epfl/bluebrain/nexus/cli/ClientError.scala

```diff
@@ -13,6 +15,31 @@ sealed trait ClientError extends Product with Serializable {
 
 object ClientError {
 
+  /**
+    * Attempt to construct a [[ClientError]] from the passed HTTP status code and message
+    *
+    * @return Some(clientError) if the code is not Successful, false otherwise
+    */
+  def apply(code: Status, message: String): Option[ClientError] =
+    Try(unsafe(code, message)).toOption
+
+  /**
+    * Construct a [[ClientError]] from the passed HTTP status code and message
+    *
+    * @throws IllegalArgumentException if the code is a Successful HTTP Status code is 2xx
+    */
+  def unsafe(code: Status, message: String): ClientError =
+    code.responseClass match {
+      case Status.Successful =>
+        "".toBoolean
```

Comment:
?

Reply:
Hahaha I don't know what that was doing there. Removed